README.md

```diff
@@ -187,6 +193,22 @@ which would result in the user's email or phone number being marked as used by t
 In order to avoid that, if you register twice with the same credentials, without activitating the first time, the system
 will accept the second registration and delete the first one.
 
+## Create a new user by using an existing user account
+
+One user connected through the backend client is allowed to create other users.
+
+```
+echo '
+{
+    "email": "TEST@EXAMPLE.COM" ,
+    "username" : "USER_NAME",
+    "plain_password" : "PLAIN_TEXT_PASSWORD",
+    "roles": ["ROLE_1", "ROLE_2"]
+}
+' |  http POST http://127.0.0.1:8089/app_dev.php/user/add 'Authorization:Bearer {accessToken}'
```

Comment:
also you should tell what this api returns (at least a json with the id of the created user , so we can then do something with this user )